cv_front_using_helper_windows: true
cv_down_using_helper_windows: false
reception_rate: 10
image_subscriber_buffer_size: 1
# This variable is used to indicate that the front cv node should be looking for the specified object.
cv_front_detect_object: "gate"
# This variable is used to indicate that the down cv node should be looking for the specified object.
cv_down_detect_object: "line"
